packages/engine/lib/simulation_structure/src/lib.rs

```diff
@@ -1,45 +1,21 @@
-use core::fmt;
-use std::str::FromStr;
-
-use serde::{Deserialize, Serialize};
-use uuid::Uuid;
-
-pub type SimulationShortId = u32;
-
-#[derive(Debug, Copy, Clone, PartialEq, Eq, Hash, Serialize, Deserialize)]
-#[serde(transparent)]
-pub struct ExperimentId {
-    id: Uuid,
-}
-
-impl ExperimentId {
-    pub fn generate() -> Self {
-        Self { id: Uuid::new_v4() }
-    }
-
-    pub fn as_bytes(&self) -> &[u8; core::mem::size_of::<u128>()] {
-        self.id.as_bytes()
-    }
-}
-
-impl fmt::Display for ExperimentId {
-    fn fmt(&self, fmt: &mut fmt::Formatter<'_>) -> fmt::Result {
-        self.serialize(fmt)
-    }
-}
-
-impl From<ExperimentId> for Uuid {
-    fn from(id: ExperimentId) -> Self {
-        id.id
-    }
-}
-
-impl FromStr for ExperimentId {
-    type Err = uuid::Error;
-
-    fn from_str(uuid_str: &str) -> Result<Self, Self::Err> {
-        Ok(Self {
-            id: Uuid::from_str(uuid_str)?,
-        })
-    }
-}
+//! Contains the structure for running the engine.
```

Comment:
Perhaps something like this
```suggestion
//! Contains the components used to define the structure of Experiment and Simulation runs.
```

That does make me wonder if we actually want to call these `experiment_structure` and `experiment_control`... Not sure

I guess it's kind of "simulation" in general instead of "_a_ simulation"

Reply:
I was also thinking about that. I'd leave that to a follow-up PR when the crate is finished.